test_scripts/Smoke/API/026_GetVehicleData_PositiveCase.lua

```diff
@@ -0,0 +1,204 @@
+  },
+  odometer = 8888,
+  beltStatus = {
+    driverBeltDeployed = "NOT_SUPPORTED",
+    passengerBeltDeployed = "YES",
+    passengerBuckleBelted = "YES",
+    driverBuckleBelted = "YES",
+    leftRow2BuckleBelted = "YES",
+    passengerChildDetected = "YES",
+    rightRow2BuckleBelted = "YES",
+    middleRow2BuckleBelted = "YES",
+    middleRow3BuckleBelted = "YES",
+    leftRow3BuckleBelted = "YES",
+    rightRow3BuckleBelted = "YES",
+    leftRearInflatableBelted = "YES",
+    rightRearInflatableBelted = "YES",
+    middleRow1BeltDeployed = "YES",
+    middleRow1BuckleBelted = "YES"
+  },
+  bodyInformation = {
+    parkBrakeActive = true,
+    ignitionStableStatus = "MISSING_FROM_TRANSMITTER",
+    ignitionStatus = "UNKNOWN"
+  },
+  deviceStatus = {
+    voiceRecOn = true,
+    btIconOn = true,
+    callActive = true,
+    phoneRoaming = true,
+    textMsgAvailable = true,
+    battLevelStatus = "ONE_LEVEL_BARS",
+    stereoAudioOutputMuted = true,
+    monoAudioOutputMuted = true,
+    signalLevelStatus = "TWO_LEVEL_BARS",
+    primaryAudioSource = "USB",
+    eCallEventActive = true
+  },
+  driverBraking = "NOT_SUPPORTED",
+  wiperStatus = "MAN_LOW",
+  headLampStatus = {
+    lowBeamsOn = true,
+    highBeamsOn = true,
+    ambientLightSensorStatus = "NIGHT"
+  },
+  engineTorque = 555.5,
+  accPedalPosition = 55.5,
+  steeringWheelAngle = 555.5,
+  eCallInfo = {
+    eCallNotificationStatus = "NORMAL",
+    auxECallNotificationStatus = "NORMAL",
+    eCallConfirmationStatus = "NORMAL"
+  },
+  airbagStatus = {
+    driverAirbagDeployed = "NOT_SUPPORTED",
+    driverSideAirbagDeployed = "NOT_SUPPORTED",
+    driverCurtainAirbagDeployed = "NOT_SUPPORTED",
+    passengerAirbagDeployed = "NOT_SUPPORTED",
+    passengerCurtainAirbagDeployed = "NOT_SUPPORTED",
+    driverKneeAirbagDeployed = "NOT_SUPPORTED",
+    passengerSideAirbagDeployed = "NOT_SUPPORTED",
+    passengerKneeAirbagDeployed = "NOT_SUPPORTED"
+  },
+  emergencyEvent = {
+    emergencyEventType = "NO_EVENT",
+    fuelCutoffStatus = "NORMAL_OPERATION",
+    rolloverEvent = "NO_EVENT",
+    maximumChangeVelocity = 0,
+    multipleEvents = "NO_EVENT"
+  },
+  clusterModeStatus = {
+    powerModeActive = true,
+    powerModeQualificationStatus = "POWER_MODE_UNDEFINED",
+    carModeStatus = "TRANSPORT",
+    powerModeStatus = "KEY_OUT"
+  },
+  myKey = {
+    e911Override = "NO_DATA_EXISTS"
+  }
+}
+
+local paramsList = {}
+local requestParams = { }
+for k, _ in pairs(vehicleDataValues) do
+  table.insert(paramsList, k)
+  requestParams[k] = true
+end
+
+local allParams = {
+  requestParams = requestParams,
+  responseUiParams = vehicleDataValues,
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFunc(tbl)
+  local SVDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+        --For now commented because SDL does not process successfully list of parameters with more then 24 items
```

Comment:
@GetmanetsIrina you could just omit list of parameters - then all parameters for this RPC will be allowed

Reply:
@AKalinich-Luxoft , Done in 9c0827d09fcbc036e9df4633cac640738392824d